src/main/resources/locale_fr_FR.properties

```
@@ -0,0 +1,196 @@
+command.repSettings.sub.reactions.true=Le robot traitera les réactions.
+command.reputation.description=Informations sur ta réputation ou celle des autres.
+command.reputation.profile.nextLevel=Niveau suivant
+command.reputation.profile.title=Réputation de %NAME%
+command.reputation.sub.top=Obtenir la liste de tête ou une page de la liste
+command.reputation.sub.top.ranking=Ranking of %GUILD% (Classement de %GUILD%)
+command.roles.description=Gère les rôles de réputation.
+command.roles.sub.add=Ajouter un rôle de réputation.
+command.roles.sub.add.added=Ajouté le rôle %ROLE% avec la réputation %POINTS%.
+command.roles.sub.list=Lister les rôles de réputation.
+command.roles.sub.managerRole=Change le rôle de gestionnaire. Les utilisateurs ayant ce rôle peuvent gérer le bot.
+command.roles.sub.managerRole.current=Le rôle de manager actuel est %ROLE%.
+command.roles.sub.managerRole.noRole=Aucun rôle de gestionnaire n'est défini.
+command.roles.sub.managerRole.set=Rôle de gestionnaire défini sur %ROLE%.
+command.roles.sub.remove=Remplacer un rôle de réputation.
+command.roles.sub.remove.notARepRole=Ce rôle n'est pas un rôle de réputation.
+command.roles.sub.remove.removed=Remove le rôle %ROLE%.
+command.scan.canceled=Scanner annulé.
+command.scan.canceling=Tentative d'annulation du scan.
+command.scan.completed=Scan terminé.
+command.scan.description=Scanner un canal pour les réputations.
+command.scan.error.history=Permission manquante pour l'historique des messages.
+command.scan.error.queueFull=Trop de scans sont en cours d'exécution. Veuillez attendre qu'un emplacement soit libre.
+command.scan.error.running=Une tâche est déjà en cours sur cette guilde. Attends qu'elle soit terminée.
+command.scan.progress=Progression : %PERCENT%
+command.scan.result=Messages scannés %SCANNED%.\nAjouté %HITS% de réputation.
+command.scan.scheduling=Planifie ton analyse. Le temps nécessaire estimé jusqu'à l'achèvement est de : %DURATION%.
+command.scan.sub.cancel=Annule un scan en cours.
+command.scan.sub.cancel.noTask=Aucune tâche n'est en cours pour cette guilde.
+command.setup.description=Exécute une configuration de base du robot.
+command.setup.dialog.channels=Veuillez saisir tous les canaux où les utilisateurs sont autorisés à recevoir de la réputation.
+command.setup.dialog.continueToProceed=Veuillez appuyer sur "$word.continue$" pour continuer.
+command.setup.dialog.loadDefaults=Charge les mots de remerciement par défaut pour tes langues.
+command.setup.dialog.managerRole=Veuillez écrire un rôle qui doit être autorisé à modifier les paramètres du robot.
+command.setup.dialog.roles=Veuillez définir des rôles de réputation.\nCes rôles seront attribués à un utilisateur lorsqu'il atteindra le niveau de réputation spécifié.\nDéfinis-les comme suit : <$words.role$> <$words.reputation$>.\nLorsque tu as défini tous tes rôles, appuie sur le bouton.
+command.setup.dialog.rolesFormat=Veuillez les définir comme <$mots.rôle$> <$mots.réputation$>.
```

Keys have been translated
```suggestion
command.setup.dialog.rolesFormat=Veuillez les définir comme <$words.role$> <$words.reputation$>.
```